pallets/stateful-storage/README.md

```diff
@@ -0,0 +1,61 @@
+# Stateful Storage Pallet
+
+The Stateful Storage pallet provides per-MSA functionality for reading and writing stateful data (where only the latest state is relevant).
+
+## Summary
+
+When a Schema uses `Paginated` or `Itemized`, the payload data is stored in this pallet.
+Data stored is user-centric instead of time-centric as with the Messages Pallet.
```

Comment:
Can we link to the messages pallet?

Reply:
Not yet. As this is part of the documentation that will be pulled into docs.frequency.xyz. It is in the list to figure out however: https://github.com/frequency-chain/docs/issues/59